test/itkLandmarkAtlasSegmentationFilterTest.cxx

```diff
@@ -75,7 +75,11 @@ readSlicerFiducials(std::string fileName)
     throw itk::ExceptionObject(
       __FILE__, __LINE__, "Fiducials file with IJK coordinates is not supported", __FUNCTION__);
   else
-    throw itk::ExceptionObject(__FILE__, __LINE__, "Unrecognized coordinate system", __FUNCTION__);
+  {
```

Comment:
The second line of the test inputs has it:
```text
# CoordinateSystem = 0
```
Slicer had written this for a very long time, if not since this file format's introduction.

Reply:
Yes, it looks like the `# CoordinateSystem = 0` line is accounted for here:

https://github.com/KitwareMedical/HASI/blob/adba8c808c0444e93d99e80c8a3e89c329e9b57e/test/itkLandmarkAtlasSegmentationFilterTest.cxx#L67

Reply:
My point is, we should never enter the `else` which throws the exception.

Reply:
I think that `Unrecognized coordinate system` is the result of using `.fcsv` file written on Windows, but running on *nix.

Maybe we should not treat `.fcsv` files as large binaries, and just directly commit them to the repository? Or regenerate them on Linux, and add to Girder?

Reply:
I don't think it is a Windows / *nix or binary content link issue -- opening the file in a text editor, it does not contain a line with *RAS*, *LPS* or *IJK*.

```
# Markups fiducial file version = 4.10
# CoordinateSystem = 0
# columns = id,x,y,z,ow,ox,oy,oz,vis,sel,lock,label,desc,associatedNodeID
vtkMRMLMarkupsFiducialNode_50,-4.945,-9.225,46.011,0.000,0.000,0.000,1.000,1,1,0,CoFH,Center of Femur Head,vtkMRMLScalarVolumeNode2
vtkMRMLMarkupsFiducialNode_51,-7.424,-9.474,44.711,0.000,0.000,0.000,1.000,1,1,0,FS,Femur Shaft,vtkMRMLScalarVolumeNode2
vtkMRMLMarkupsFiducialNode_52,-5.018,-9.058,47.535,0.000,0.000,0.000,1.000,1,1,0,D,Dent,vtkMRMLScalarVolumeNode2
```

Reply:
Ah -- I see the code is also checking for *0*, *1*, or *2* as the "last" character. Perhaps we should change this to check to see if the line contains the numerical character?

Reply:
End-user text editors usually support CRLF even on Linux. Can you check line endings in the text editor and try converting into LF?